linux/user-and-file-management/linux-intro/what-is-linux.md

```diff
@@ -0,0 +1,39 @@
+---
+author: kapnobatai136
+
+aspects:
+  - introduction
+
+type: normal
+
+category: must-know
+
+tags:
+  - introduction
+  - linux
+  - operating-system
+
+---
+
+# What's Linux?
+
+---
+## Content
+
+To understand what Linux is, let's go back to the very basics of computing.
+
+To complete even the most basic operation, something must facilitate the communication between you and the hardware. 
+
+That's what an **operating system** does. It makes it possible for you to read this.
+
+Along with Windows, Android, or MacOS, Linux is an **operating system**. You can find it in many places. From your computer, to your car and refrigerator, there's a high probability that they run on Linux.
```

Comment:
How about?

```suggestion
A computer consists of physical parts, called *hardware*, and written programs, called *software*. 

Most computers have a special program that manages all other programs and helps them use the underlying hardware.

That program is called an **operating system**, or **OS** for short.

Although commercial computers that most of us use every day are mainly using Windows or MacOS, Linux is by far the most versatile OS.

From cloud storage and space rockets, to your car or refrigerator, there's a high chance that they all run on Linux.

[what-are-clouds-made-of](https://img.enkipro.com/e763d343e3f748f707da85221e99d44d.png)
```

Reply:
The reason we should mention Unix here is because it's core to the Linux history and it saves us from any potential future mentions :)